File: C2203L/Java2/07-12-2022/musicapp/src/main/java/com/aptech/Artist.java
package com.aptech;

public class Artist {
    private String name;

    public Artist() {}
    public Artist(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "Artist{" +
                "name='" + name + '\'' +
                '}';
    }
}

File: A2209I/DMA/ex003/Models/Artist.cs
using Microsoft.Extensions.Hosting;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace ex003.Models
{
    [Table("Artists")]
    public class Artist
    {
        [Key]
        public int ArtistId { get; set; }
        [Required]
        [MaxLength(200, ErrorMessage ="Length must be <= characters")]
        public string Name { get; set; }
        public string Bio { get; set; }

        [JsonIgnore]
        public virtual ICollection<Artwork> Artworks { get; set; } = new List<Artwork>();
    }
}
